internal/cephfs/volumeoptions.go

```diff
@@ -373,10 +385,18 @@ func newVolumeOptionsFromStaticVolume(volID string, options map[string]string) (
 	// store NOT of static boolean
 	opts.ProvisionVolume = !staticVol
 
-	opts.ClusterID, opts.Monitors, opts.SubvolumeGroup, err = getClusterInformation(options)
+	clusterData, err := getClusterInformation(options)
+
 	if err != nil {
 		return nil, nil, err
 	}
+	if clusterData == nil {
```

Comment:
I think it is safe to assume that `clusterData` is not `nil` if there was no error. No need to have this if-statement for a 2nd error check.

Reply:
Okay.